### ISSUE TITLE
Google home states if can not reach homebridge

### ISSUE THREAD

**Issue by sterling90x** [oznu/homebridge-gsh#154]
**Describe Your Problem:**
Homebridge google home plugin has been working flawlessly, but recently google home is stating it can not reach homebridge. 

I've tried restarting homebridge and unlinking and linking homebridge in google home.

**Comment by gurselm** [comment]
Same for me

**Comment by akagarwal** [comment]
I have the same issue

**Comment by marcgrc** [comment]
GSH server DOWN.
News?

**Comment by heggink** [comment]
It's been on and off. Reinstalled yesterday and then it works again. This morning it was gone again.

**Comment by freeforall2** [comment]
In my case the GSH plug-in states that it can’t reach Homebridge whereas in reality Homebridge is working. Other plugins such as the one for Siri is working perfectly. However in my case I can fix the issue each time it happens by restarting Homebridge through the admin interface. Which is painful

**Comment by jlgore** [comment]
Sorry I don't have more information, but I noticed I was having this problem. Unrelated (or so I thought) to this problem I ended up having to re-connect / re-authenticate to the cloud platform by adding homebridge to my Google Home again. After doing that my connection seems to be much more stable. Wish I had some more detailed info to provide. Maybe a coincidence but I noticed a pretty immediate change after homebridge restarted.
